CONTRIBUTING.md

```diff
@@ -62,6 +62,11 @@ The new tag will trigger the CI (`release.yml`) that will build and release the
 
 Release notes can then be written in the [Releases page](https://github.com/silx-kit/h5grove/releases).
 
+The following commands can be used to handle the pre-release cycle:
+- To tag the first pre-release, run `bump2version --new-release <X.Y.Zb0> [patch|minor|major]`
```

Comment:
```suggestion
- To tag the first pre-release, run `bump2version --new-version <X.Y.Zb0> [patch|minor|major]` where `X.Y.Z` is the version number of the next release
```

Note that when using `new-version`, you can put whatever you want instead of `[patch|minor|major]`. But it has to be there (https://github.com/c4urself/bump2version/issues/22)

Reply:
Good to know!